pkg/utils/docker/image.go

```diff
@@ -0,0 +1,42 @@
+package docker
+
+import (
+	"os"
+	"regexp"
+	"strings"
+)
+
+// ParseDockerImage extracts repo and tag from image. An empty string is returned if no tag is discovered.
+func ParseDockerImage(image string) (name string, tag string) {
+	matches := strings.Split(image, ":")
+	if len(matches) >= 2 {
+		return matches[0], matches[1]
+	} else if len(matches) == 1 {
+		return matches[0], "latest"
+	}
+	return "", ""
+}
+
+// GetImageRepoTagFromEnv parse the image and tag from environment varaibles, if it's not existed or
```

Comment:
or?

Reply:
Updated.

Reply:
if it's not existed, will use default image and image tag.